manifests/config_nginx.pp

```diff
@@ -18,9 +18,7 @@
 
   package {
     'nginx':
-      ensure => installed,
-      before => Exec['Chown graphite for web user'],
-      notify => Exec['Chown graphite for web user'];
```

Comment:
This notifies the chown, which has a 'refreshonly'. So the chow is done only the first time nginx is installed